app/frontend/src/components/ApplicationList.vue

```diff
@@ -45,23 +49,26 @@
 import { mapActions, mapGetters } from 'vuex';
 
 import { KcEnv } from '@/utils/constants';
-import { buildClientStatusSpan } from '@/utils/util.js';
+import ClientStatus from '@/components/apiAccess/ClientStatus.vue';
 
 export default {
   name: 'ApplicationList',
+  components: {
+    ClientStatus
+  },
+  data() {
+    return {
+      envs: KcEnv
```

Comment:
This should be a computed - especially if it's supposed to be a constant resource.

Reply:
I guess